src/Pluralize.js

```diff
@@ -7,11 +7,19 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { pluralize } from './utils'
 
-const Plural = ({ className, style, ...props }) => (
-  <span className={className} style={style}>
-    {pluralize(props)}
-  </span>
-)
+const Plural = ({ className, style, ...props }) => {
+  return (
+    className || style
+      ? (
+        <span className={className} style={style}>
+          {pluralize(props)}
+        </span>
+      ) : (
+        <React.Fragment>
+          {pluralize(props)}
+        </React.Fragment>
```

Comment:
Don't think `Fragments` are required here - can you tidy this up?

Reply:
Cool. Let me do so.